### PR TITLE
Change headings to sentence case and replace &

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/contribution-guide.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ you can use the `children` shortcode to show all of the sub-topics for the secti
 {{%/* children style="h2" description="true" */%}}
 ```
 
-## Writing Article Content
+## Writing article Content
```
Comment (banker) [inline content/contribution-guide.md:93]: "content" should be lower case

File: content/platforms/kubernetes/upgrading-kubernetes-with-operator.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ To see the status of the current rolling upgrade, run:
 kubectl rollout status sts <REC_name>
 ```
 
-## How Does the Upgrade Work?
+## How does the Upgrade Work?
```
Comment (banker) [inline content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:55]: Not sentence case.

File: content/platforms/release-notes/k8s-5-4-10-8-january-2020.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ For legacy Kubernetes versions, deployment files are available in the documentat
 
 - This new release deprecates the Redis Enterprise Operator support for legacy Kubernetes version 1.8.
 
-## Known Limitations
+## Known limitations
 
-### Custom Resource Definition Changes
+### Custom resource Definition Changes
```
Comment (banker) [inline content/platforms/release-notes/k8s-5-4-10-8-january-2020.md:55]: not sentence case

File: content/rc/_index.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ With Redis Cloud, we automatically add or remove instances to your cluster
 to accommodate your fluctuating dataset size without any interruption
 to your applications.
 
-### Significant Cost Saving of Redis on Flash (RoF)
+### Significant cost saving of Redis on Flash (RoF)
```
Comment (banker) [inline content/rc/_index.md:44]: s/saving/savings

File: content/rc/administration/configuration/monitoring-performance.md [hunk 5]
```diff
@@ -81,7 +81,7 @@ appropriately on your own. For example, if you have two databases,
 configuring an alert for when each database reaches 80% can surpass your
 number of subscriptions.
 
-## Changing Alert Email Recipients
+## Changing alert Email Recipients
```
Comment (banker) [inline content/rc/administration/configuration/monitoring-performance.md:84]: not sentence case

File: content/rc/administration/security/_index.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ aliases: /rv/administration/configuration/securing-your-database/
 ---
 The security controls for your database are:
 
-## Access Control by Source IP/Subnet
+## Access control by source IP/subnet
```
Comment (banker) [inline content/rc/administration/security/_index.md:15]: "source IP or subnet"

File: content/rc/administration/security/data-access-control.md [hunk 7]
```diff
@@ -89,7 +89,7 @@ To assign Redis ACLs to a data access role:
 
 Users that are assigned to the role can access the databases according to the Redis ACL definitions.
 
-## Assigning Users to a User Role
+## Assigning users to a User Role
```
Comment (banker) [inline content/rc/administration/security/data-access-control.md:92]: not sentence case

File: content/rc/administration/security/securing-redis-cloud-connections.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ aliases: /rv/securing-redis-cloud-connections/
 In Redis Cloud, SSL/TLS support is a paid option. To enable it, contact [Support](https://support.redislabs.com).
 SSL/TLS support is included in all Redis Cloud Pro paid plans.
 
-## Setting Up Your Database
+## Setting up Your Database
```
Comment (banker) [inline content/rc/administration/security/securing-redis-cloud-connections.md:13]: not sentence case

File: content/rc/administration/security/securing-redis-cloud-connections.md [hunk 9]
```diff
@@ -280,7 +280,7 @@ delay = yes`
 accept = 127.0.0.1:6379
 connect = host:port`
 
-### Testing Secure Connectivity to a Redis Cloud Resource
+### secure connectivity to a Redis Cloud resource
```
Comment (banker) [inline content/rc/administration/security/securing-redis-cloud-connections.md:283]: not sentence case

**Comment by bmansheim** [comment]
We still need to do page titles.

**Pull request merged**